packages/consumption/src/consumption/ConsumptionController.ts

```diff
@@ -87,7 +89,12 @@ export class ConsumptionController {
         requestItemProcessorOverrides = new Map<RequestItemConstructor, RequestItemProcessorConstructor>(),
         notificationItemProcessorOverrides = new Map<NotificationItemConstructor, NotificationItemProcessorConstructor>()
     ): Promise<ConsumptionController> {
-        this._attributes = await new AttributesController(this, this.transport.eventBus, this.accountController.identity).init();
+        this._attributes = await new AttributesController(
+            this,
+            this.transport.eventBus,
+            this.accountController.identity,
+            this.consumption.config.setDefaultRepositoryAttributes
```

Comment:
Not sure if I should do it like this or hand down `config` as a whole.

Reply:
In general I am not a big fan of boolean parameters, but with enabled inlay hints it's ok here.